src/components/NavBar.js

```diff
@@ -1,12 +1,49 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React from "react";
+import { Link } from "react-router-dom";
 
-const NavBar = () => (
-  <nav className="nav-wrapper grey darken-3">
-    <div className="container">
-      <Link to="/" className="brand-logo" />
-    </div>
-  </nav>
-);
+function NavBar() {
```

Comment:
Looks way too complicated, DRY not being followed, Link component shouldn't be removed, A element should be avoided if possible

Reply:
I dont know what are you talking about.